### ISSUE TITLE
Permissions on tmp folder

### ISSUE THREAD

**Issue by timothystewart6** [techno-tim/k3s-ansible#69]
Need to verify this issue but someone in Discord said that they had an error  when running the playbook.  The permissions on the folder for the first master were too strict and it could not apply the manifest for metal lb.  They had to chown the directory to their ansible user.

We should also test that this folder is removed by ansible with the fix (verify it's gone)

## Expected Behavior

should complete without error

## Current Behavior

Errors out and you need to chown the tmp folder

## Steps to Reproduce

1. run the playbook
2. error when deleting tmp folder.

## Context (variables)

Operating system:

ubuntu 22.04.1 cloud-init image

## Possible Solution

- [x] I've checked the [General Troubleshooting Guide](https://github.com/techno-tim/k3s-ansible/discussions/20)

**Comment by derekadair** [comment]
Just to clarify, no issues deleting the dir. The first master instance wouldn't start up because it got a permission denied accessing the metallb yaml in that directory (which was also created without problems). Had to chown the directory to my ansible user to progress.

**Comment by timothystewart6** [comment]
Thank you @derekadair !  I updated the description!

**Comment by timothystewart6** [comment]
Bug is probably right here https://github.com/techno-tim/k3s-ansible/blob/master/roles/k3s/post/tasks/main.yml#L4

**Comment by timothystewart6** [comment]
should probably use the user from a variable and then verify it's gone after we delete it

**Comment by timothystewart6** [comment]
This was closed by https://github.com/techno-tim/k3s-ansible/commit/4365a2a54b6eba3058282a3b4b4bb489e764e149  Sorry, I pushed instead of opening an PR!

**Comment by gbyczyns** [comment]
I think that you cannot assume that the group name will be always equal to user name. In my case, those two were different and, as a result, I got an error: `chgrp failed: failed to look up group <user_name_here>`.

I had to remove this line completely (there are two occurrences) to make it work again:
`    group: "{{ ansible_user }}"`
Ansible documentation says that "When left unspecified, it uses the current group of the current user".
